pkg/apiserver/rest/apis/v1/types.go

```diff
@@ -292,14 +291,15 @@ func (e EnvBindingList) ContainTarget(name string) bool {
 
 // ApplicationBase application base model
 type ApplicationBase struct {
-	Name        string            `json:"name"`
-	Alias       string            `json:"alias"`
-	Project     *ProjectBase      `json:"project"`
-	Description string            `json:"description"`
-	CreateTime  time.Time         `json:"createTime"`
-	UpdateTime  time.Time         `json:"updateTime"`
-	Icon        string            `json:"icon"`
-	Labels      map[string]string `json:"labels,omitempty"`
+	Name         string            `json:"name"`
+	Alias        string            `json:"alias"`
+	Project      *ProjectBase      `json:"project"`
+	Description  string            `json:"description"`
+	CreateTime   time.Time         `json:"createTime"`
+	UpdateTime   time.Time         `json:"updateTime"`
+	Icon         string            `json:"icon"`
+	Labels       map[string]string `json:"labels,omitempty"`
+	WebhookToken string            `json:"webhookToken"`
```

Comment:
have only one Token?

Reply:
It is recommended to provide an independent webhook query API,  we should implement deletion and creation API in the follow-up.

Reply:
A application should have multiple webhooks?
The webhook is an independent API